### PR TITLE
English translations for BWL scripts

### PR DESCRIPTION
**Pull request by henkethebenke**

### CODE REVIEW

**Comment by sidsukana** [comment]
Let the games begin!

File: src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_vaelastrasz.cpp [hunk 1]
```diff
@@ -59,20 +59,20 @@ enum
...
 
-    // TO FIX : l'effet chaine.
+    // TO FIX : The chain effect
     // DB : spell_effect_mod.effectChainTarget = 25;
     // Cleave attack that hits for 2k. This is a chain cleave, so if positioning is poor it can chain to the entire raid, even to behind him. It is critical that nobody be within approximately 10 yards of the main tank for this reason. Offtanks should be generating threat far enough from the MT to avoid chaining the cleave, but close enough to slide into place when BA hits the MT.
-    SPELL_CLEAVE                = 22540, // 22540 Chain Cleave hardcodé via la BD (Ustaag) // 19983 // 20684  ? // Chain cleave is most likely named something different and contains a dummy effect
-    // changer spell.cpp à "//FIXME: This very like horrible hack and wrong for most spells"
-    // chercher quelles créatures utilisent les deux spells dans la DB ?
+    SPELL_CLEAVE                = 22540, // 22540 Chain Cleave hardcoded via the BD (Ustaag) // 19983 // 20684  ? // Chain cleave is most likely named something different and contains a dummy effect
+    // change spell.cpp à "//FIXME: This very like horrible hack and wrong for most spells"
```
Comment (Rockette) [inline src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_vaelastrasz.cpp]: Remove the "à"?

File: src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_victor_nefarius.cpp [hunk 2]
```diff
@@ -329,20 +329,20 @@ struct boss_victor_nefariusAI : ScriptedAI
...
             {
                 pNefarian->CastSpell(pNefarian, SPELL_HOVER, true);
                 pNefarian->SetFly(true);
             }
 
-            // 40 tués il pop
-            // 42 tués fin de pop des adds
-            // il vole puis arrivé son spawn point crie et reste en l'air
-            // 10 sec plus tard : se pose puis cast shadow flame
+            // 40 drakes killed
```
Comment (Rockette) [inline src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_victor_nefarius.cpp]: Shouldn't be "Nefarian spawn when 40 drakes are killed"?

Comment (keyndin) [inline src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_victor_nefarius.cpp]: "After killing 42 Drakonids, no more will spawn" (Quoted from http://wowwiki.wikia.com/wiki/Nefarian_(tactics))

**Pull request merged**